content/word-lists/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Overview"
+---
+
+# Language recommendation lists 
+
+The Inclusive Naming Initiative produces 3 tiers of language recommendation. These are compiled into lists for easy consumption by adopters. Terms are separated into lists based on the severity of the term in question as well as the level of review the terms have received.
+
+## Tier 1: Replace Immediately
+
+Tier 1 terms should be replaced whenever encountered. 
+Terms included in this list have one or all of the following: 
+
+- Strong social consensus among the software development community on replacements
+- Are identified by the Inclusive Naming Initiative as high severity terms in need of immediate replacement 
+- Terms where the impact of changing or removing is low: there is little entanglement in low-level systems or standardized language set by standards bodies, for example
```

Comment:
This is arguably wrong as written, e.g. whitelist and blacklist are unfortunately ubiquitous. Yet, they are relatively straightforward and low-impact to change.

Reply:
1. "high severity" is an adjectival phrase, so should be hyphenated.
2. "changing or removing is low" → "change or removal", so as to avoid leaving dangling verbs with no subject.
3. "for example, there is little entanglement" — put "for example" up front so the reader does not get a surprise at the end of the sentence.